### ISSUE TITLE
is this normal or a bug idk

### ISSUE THREAD

**Issue by MIMIIZ2** [patientx/ComfyUI-Zluda#123]
### Your question

D:\games\ComfyUI-Zluda\comfy\ldm\modules\attention.py:448: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)

also these every generate if i close webui and launch then after 1 generate it becomes faster and not doing it 

### Logs
```powershell
FETCH ComfyRegistry Data: 35/83
FETCH ComfyRegistry Data: 40/83
FETCH ComfyRegistry Data: 45/83
FETCH ComfyRegistry Data: 50/83
FETCH ComfyRegistry Data: 55/83
FETCH ComfyRegistry Data: 60/83
FETCH ComfyRegistry Data: 65/83
FETCH ComfyRegistry Data: 70/83
FETCH ComfyRegistry Data: 75/83
FETCH ComfyRegistry Data: 80/83
```

**Comment by patientx** [comment]
that is an annoying database update of comfy, no its not avoidable. People asked in the comfyui github. It needs to do it everytime , yes.
